free-1.md

```diff
@@ -10,34 +13,75 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Legal writing is both notoriously complex and requires a lot of context to interpret.
+The users of the language would probably be lawyers, judges, lawmakers or reporters who wanted
+to dig into the context, history and use of various laws, definitions, rulings and interpretations.
```

Comment:
Not laypeople? I feel like the people you mentioned already have the resources to understand legal writing but the standard user signing an EULA or trying to figure out if they actually broke a traffic law could use help here